packages/contracts-bedrock/test/L1/SuperchainConfig.t.sol

```diff
@@ -149,31 +149,23 @@ contract SuperchainConfig_AddDependency_Test is CommonTest {
         superchainConfig.addDependency(_chainId, _systemConfig);
     }
 
-    /// @notice Tests that `addDependency` reverts when the chain is already in the dependency set.
-    function test_addDependency_chainAlreadyExists_reverts(uint256 _chainId) external {
-        vm.assume(_chainId != block.chainid);
-
-        vm.startPrank(superchainConfig.dependencyManager());
-        superchainConfig.addDependency(_chainId, address(systemConfig));
-
-        vm.expectRevert(SuperchainConfig.DependencyAlreadyAdded.selector);
-        superchainConfig.addDependency(_chainId, address(systemConfig));
-        vm.stopPrank();
-    }
-
     /// @notice Tests that `addDependency` reverts when the dependency set is too large.
     function test_addDependency_dependencySetTooLarge_reverts() external {
         vm.startPrank(superchainConfig.dependencyManager());
 
+        // Add the maximum number of dependencies to the dependency set
         uint256 i;
         for (i; i < type(uint8).max; i++) {
             superchainConfig.addDependency(i, address(systemConfig));
         }
```

Comment:
NIT: No need to apply this, but just letting you know:
You can declare `i` inside the for loop:
`for (uint256 i; i < type(uint8).max; i++) {`

Reply:
true